Dockerfile

```diff
@@ -20,4 +21,4 @@ COPY --from=builder /app/ ./
 
 EXPOSE 8080
 
-CMD ["yarn", "server"]
\ No newline at end of file
+CMD ["npm run", "server"]
```

Comment:
I _think_ "npm" and "run" should be separated out via separate comma/quotes, but it looks like it built fine so maybe not